base/_buttons.scss

```diff
@@ -0,0 +1,136 @@
+$button-border: 4px;
+$button-background-hover: rgba($theme-light-border, .1);
+$button-background-hover--disabled: rgba($theme-light-border, .3);
+$button-background--loading: rgba($theme-light-border, .4);
+
+.go-button {
+  @include button-border-gradient;
+
+  background-origin: padding-box, border-box; // This is important
+  background-repeat: no-repeat; // This is important
+  border: $button-border solid transparent;
+  border-radius: $global-radius;
+  color: $base-dark;
+  cursor: pointer;
+  font-family: $heading-font-stack;
+  font-size: 1rem;
+  line-height: 1rem;
+  outline-color: $theme-light-border;
+  outline-offset: -2px;
+  padding: $column-gutter--half $column-gutter;
+  position: relative;
+  text-transform: uppercase;
+  transition: color $global-transition-duration $global-transition-timing;
+
+  &::before {
+    border: ($button-border / 2) solid $base-light;
+    border-radius: $global-radius;
+    bottom: -1px;
+    content: '';
+    left: -1px;
+    position: absolute;
+    right: -1px;
+    top: -1px;
+    transition: background $global-transition-duration $global-transition-timing;
+  }
+
+  &:hover,
+  &:disabled {
+    color: $base-darkest;
+  }
+
+  &:hover:not(.go-button--loading)::before {
+    background: $button-background-hover;
+  }
+
+  &:disabled:not(.go-button--loading) {
+    @include button-border-gradient;
+
+    cursor: not-allowed;
+
+    &:hover::before,
+    &::before {
+      background: $button-background-hover--disabled;
+    }
+
+    .go-button__icon {
+      @include fill-text-background($color: $theme-light-border);
+    }
+  }
+
+  .go-button__icon {
+    font-size: 1.25rem;
+    line-height: inherit;
+    margin-left: -$column-gutter--quarter;
+    margin-right: $column-gutter--quarter;
+    vertical-align: top;
+  }
+}
+
+.go-button--loading {
+  color: $base-darkest;
+  cursor: wait;
+
+  &::before {
+    animation: background-animation 1.75s linear infinite;
```

Comment:
I could see pulling it out into a variable at the top, but I don't think we should make this a global variable. This will likely be the only instance of this animation in the codebase.

Reply:
I can see the case for that, whenever we implement a linter it will yell at us if we use the same one that we have for MOBI.